packages/artifacts/README.md

```diff
@@ -0,0 +1,31 @@
+# Artifacts Package for Admin Plugin
+
+This package is responsible for generating and storing ABIs for smart contracts. It checks out the specified branch, compiles the contracts, and saves the ABIs in the `src/abis/` directory. The package can then be published to NPM for use in other projects.
```

Comment:
```suggestion
This package contains the ABIs of the OSx Admin plugin, as well as the address of the plugin repository on each network. Install it with:

```sh
yarn add @aragon/admin-plugin
```

Reply:
GitHub's diff is omitting the closing ticks ```
These would need to be added manually

Reply:
amended